clang/lib/DPCT/ErrorHandle/Error.cpp

```diff
@@ -246,6 +248,22 @@ std::string getCheckVersionFailWarning() {
          "overview.html for more "
          "details.\n";
 }
+std::string getBuildScriptNotSpecifiedWarning() {
+  return "Warning: Only CMake scripts will be migrated as no "
+         "--migrate-build-script option is provided. "
+         "See https://www.intel.com/content/www/us/en/docs/"
```

Comment:
suggest to remove the URL link.  It's not necessary.

Reply:
I'll add another PR for adding SYCLomatic doc and cherry-pick its commit to remove it on innersource.
Is that fine?